_includes/_boinc_1B.htm

```diff
@@ -64,28 +48,25 @@ <h6>
         <!--Step 3 content-->
         <div class="row">
             <div class="col-xs-12">
-                <h4 class="text-bold">Manually setting up projects</h4>
+                <h4 class="text-bold">No Account Manager Setup</h4>
                 <ul>
                     <li>
-                        After installing and starting the BOINC Manager, you are asked to add a project or use an account manager. Select the 'add a project' radio button.
-                    </li>
-                    <li>
-                        After clicking “Next“ choose a desired project.
+                        After starting BOINC, select the 'add a project' button.
                     </li>
                     <li>
-                        You will be prompted to either create an account or login to an existing one. For some projects (such as World Community Grid) you have to register directly on the homepage of the project. Remember to use the same email address for all BOINC projects.
+                        After clicking “Next“ choose a project.
                     </li>
                     <li>
-                        You will be prompted with a registration confirmation. Once you have pressed the 'Finish' button, a web browser should launch with additional registration fields required. If not, go to the project homepage (you can definitely find it via Google). You have to find the “Your Account” button and press it. The project homepage of a different project can look different, but should have the same functions!
+                        For some projects (such as World Community Grid) you have to register directly on the homepage of the project. Remember to use the same email address for all BOINC projects.
                     </li>
                     <li>
-                        Within the launched web page, enter your username (don't bother with country/post-code) then press OK.
+                        Confirm the registration. After tapping the 'Finish' button, a browser should come up. If not, go to the project homepage (can be found via Google). Find the “Your Account” button and press it.
```

Comment:
Should say "a browser _window_ ..."

Reply:
Why browser window over browser?

Reply:
To me saying "a browser should come up" sounds very awkward

Reply:
What about "a windows should come up"

Reply:
that sounds worse imo lol

Reply:
How about something like a "browser should open" or "a window should open". Browser window just feels off in some way to me. Maybe because it's not as commonly said? Not sure why it sounds off to me.

Reply:
I guess either of those would be acceptable.  @ShmoogleOsukami, thoughts?

Reply:
I think a "a browser should open" would be best between the two. Better to be specific about it.

Reply:
Done